README.md

```diff
@@ -53,15 +53,15 @@ Contributions are welcome! If you'd like to contribute to the project, Please re
 
 ## Thank you for contributing to our project!
 
- Your contributions are valuable, and we appreciate your effort in making our project better. 
- 
+Your contributions are valuable, and we appreciate your effort in making our project better.
+
  <br>
 
- **If you like the project, please feel free to give it a ⭐️**
+**If you like the project, please feel free to give it a ⭐️**
 
 ## Contributors
 
-Meet the talented individuals who have contributed to Image Searcher:
+Meet the talented individuals who have contributed to SnapGrid:
 
 <a href="https://github.com/Vijaykv5/Image-Searcher/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=Vijaykv5/Image-Searcher" />
```

Comment:
change to this : 
<a href="https://github.com/vijaykv5/snapGrid/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=vijaykv5/snapGrid" />
</a>

Reply:
<a href="https://github.com/vijaykv5/snapGrid/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=vijaykv5/snapGrid" />
</a>